modules/xmpp/SDPUtil.js

```diff
@@ -4,7 +4,11 @@ var RTCBrowserType = require("../RTC/RTCBrowserType");
 
 SDPUtil = {
     filter_special_chars: function (text) {
-        return text.replace(/[\\\/\{,\}\+]/g, "");
+        // XXX Neither null, nor undefined "contain" special chars.
+        return
+            (typeof text === 'undefined')
```

Comment:
If text is `null`, `(typeof text === 'undefined')` will be false. Maybe we should add check for `null` too.